Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -528,6 +528,7 @@ void WebProcessProxy::getLaunchOptions(ProcessLauncher::LaunchOptions& launchOpt
 #if !PLATFORM(GTK) && !PLATFORM(WPE)
```

Comment:
We define this in `AuxiliaryProcessProxy.cpp` as `#if !PLATFORM(COCOA)`; maybe we should do the same here?

Reply:
That is a good point. In this case I think we want to have this enabled for PLATFORM(COCOA), so that the method AuxiliaryProcessProxy::platformGetLaunchOptions in AuxiliaryProcessProxyCocoa.mm will be called.